python/CMakeLists.txt

```diff
@@ -14,6 +14,13 @@ endif()
 set(PYBIND11_CPP_STANDARD -std=c++17)
 add_subdirectory(pybind11)
 
+# Find path in which to install the Python module.
+find_package(PythonInterp REQUIRED)
```

Comment:
Do we need the `find_package(PythonInterp REQUIRED)`?
Can we instead ensure that we use the `PYTHON_EXECUTABLE` variable?

Reply:
How is `PYTHON_EXECUTABLE` set if we don't use `find_package(PythonInterp REQUIRED)`? Locally I see that is it set to the `python` in the path, but I don't see where or how. I thought you had to tell CMake this.